crates/ruff_dev/src/generate_docs.rs

```diff
@@ -45,6 +45,14 @@ pub(crate) fn main(args: &Args) -> Result<()> {
                 output.push('\n');
             }
 
+            if rule.is_deprecated() {
```

Comment:
I find the terminology here a bit confusing. Does this apply to rules that are in their deprecation period or only to rules that have been removed after being deprecated?

Reply:
This is actually wrong! It should be `rule.is_removed()`.